configure.ac

```diff
@@ -2511,7 +2514,11 @@ else
     EXPAND_VARIABLE(datadir, e_datarulesdir, "/suricata/rules")
     EXPAND_VARIABLE(localstatedir, e_datadir, "/lib/suricata/data")
     EXPAND_VARIABLE(ruledirprefix, e_defaultruledir, "/suricata/rules")
-
+    if test "$have_suricata_update" = "yes"; then
+        EXPAND_VARIABLE(classificationconfprefix, e_classificationconfprefix, "/suricata/rules/")
+    else
+        EXPAND_VARIABLE(classificationconfprefix, e_classificationconfprefix, "/suricata/")
+    fi
```

Comment:
I'm not sure if and where do we check whether suricata-update is bundled for Windows. That is not a part of this patch. @jasonish ?

Reply:
@pevma Do the Windows builds include Suricata-Update?

@inashivb I'm not really sure what is done in Windows for Suricata-Update.

Reply:
Currently no SU included in MSI. There is a redmine issue ticket for inclusion though. @jasonish  something we need to work on that maybe for the next major (7) release.